lib/moonshot/build_mechanism/github_release.rb

```diff
@@ -204,6 +204,7 @@ def fetch_changes(version)
     #
     # @return [String] Status and links to the CI jobs
     def check_ci_status(sha)
+      say("Checks for the commit's CI job status.", :yellow)
```

Comment:
I don't think this is required, we already say 'validating commit'

Reply:
this for double check and making sure this function not callable if flag sent

Reply:
I think we should probably drop this as its not required.

Reply:
Okay, May be change it ot 'Checking' since this is output runtime.

Reply:
Fixed